server/liveblog/themes/themes.py

```diff
@@ -81,6 +83,21 @@ def update_registered_theme_with_local_files(self):
                 created.append(theme)
         return (created, updated)
 
+    def on_delete(self, deleted_theme):
+        blogs_service = get_resource_service('blogs')
+        blogs = blogs_service.get(req=None, lookup={'theme._id': deleted_theme['_id']})
+        for blog in blogs:
+            # will assign the default theme to this blog
+            theme = blogs_service.get_theme_snapshot(blog['blog_preferences']['theme'])
+            try:
+                blogs_service.system_update(ObjectId(blog['_id']), {'theme': theme}, blog)
```

Comment:
To update the theme, change the value of `blog_preferences.theme`. The `theme` property will be updated in `on_update()`